_posts/2020-07-24-when-scenarios-and-reality-diverge/when-scenarios-and-reality-diverge.Rmd

```diff
@@ -0,0 +1,380 @@
+  ggplot2::facet_grid(rows = vars(sector), cols = vars(year_range),switch="x") + 
+  ggplot2::labs(x = "Year", y = "Ratio", title = "Figure 5: Trajectory of emission intensity reductions") + 
+  ggplot2::theme(axis.text.x = element_blank(), axis.ticks.x = element_blank(), axis.line.x = element_blank(), axis.title.x = element_blank(), strip.text.x = element_text(family="GT America", color=primary_blue, size = 10, face = "plain"), panel.spacing = unit(0.1, "cm"))
+```
+<br>
+
+Finally, Figure 6 shows the implications of applying the scenario's trajectory to the portfolio benchmark after the scenario's transition starts. The greater the disconnect between the portfolio baseline and the scenario's transition, the less ambitious the portfolio's scenario aligned target (i.e., 2060 $CO_2$ intensity).  While Figure 5 explores the temporal disconnect between the scenario and portfolio to an extreme, even a small delay, say 2014 versus 2019, impacts the scenario aligned portfolio target.
+
+```{r, fig.height=8, fig.width=8}
+etp2017_sda_targets %>% 
+  index_scenario_to_market(
+    aviation_intensity = 120,
+    steel_intensity = 2.1, 
+    cement_intensity = 1
+  ) %>% 
+  add_nice_names() %>% 
+  ggplot2::ggplot(aes(x = year, y = emission_factor_ald, group = lag, color = scenario)) +
+  ggplot2::geom_line() + 
+  ggplot2::scale_color_manual(values = c(primary_green, primary_orange, primary_blue)) +  
+  ggplot2::labs(x = "Year", y = "", title = "Figure 6: Implications on the aligned portfolio benchmark", caption = "*Each line corresponds to a lagged scenario trajectory multiplied by portfolio's current CO2 intensity.") + 
+  ggplot2::facet_grid(
+    rows = vars(sector), 
+    cols = vars(scenario),
+    switch = "y",
+    scales = "free_y", 
+    labeller = as_labeller(c(Steel = "t CO2 per t steel", Cement = "t CO2 per t cement", Aviation = "g CO2 per PKM", RTS = "RTS", B2DS = "B2DS", SDS = "SDS"))
+  ) + 
+  theme_2dii_ggplot() + 
+  theme(strip.placement = 'outside')
+```
+<br><br>
+
+Complementing Figure 6, Tables 2 and 3 show in concrete terms, the variation in the scenario aligned target in 2060. The greater the delay between the scenario's transition and the portfolio's baseline (denoted by the column Lag N), the less ambitious the emission intensity target. 
+
+```{r, echo=TRUE, warning=FALSE, fig.height=8, fig.width=8}
+table_data <- etp2017_sda_targets %>%    
+  index_scenario_to_market(
+    aviation_intensity = 120,
+    steel_intensity = 2.1, 
+    cement_intensity = 1
+  ) %>% 
+  dplyr::select(
+    scenario,
+    sector, 
+    year, 
+    emission_factor_ald, 
+    lag
+  ) %>% 
+  tidyr::pivot_wider(
+    names_from = "year",
+    values_from = "emission_factor_ald"
+  ) %>% 
+  dplyr::arrange(
+    sector,
+    scenario,
+    lag
+  ) %>% 
+  dplyr::transmute(
+    sector, 
+    scenario,
+    `Lag n` = lag, 
+    Target = ifelse(is.na(`2060`), "", round(`2060`, digits = 1))
+  ) %>% 
+  add_nice_names() %>% 
```

Comment:
Maybe show the definition of this function or use instead `janitor::clean_names()`.